book/src/keymap.md

```diff
@@ -129,6 +129,7 @@
 | `X`                   | Extend selection to line bounds (line-wise selection)             | `extend_to_line_bounds`              |
 | `Alt-x`               | Shrink selection to line bounds (line-wise selection)             | `shrink_to_line_bounds`              |
 | `J`                   | Join lines inside selection                                       | `join_selections`                    |
+| `A-J`                 | Join lines inside selection and select space                      | `join_selections_space`              |
```

Comment:
I am not sure if we want to use `A-j` or `A-J` so I just go with `A-J`.

Reply:
`A-J` makes more sense since it's an alternate behaviour of `J`.